raw_data/preprocessed/africa/liberia_english.txt

```diff
@@ -0,0 +1,376 @@
+
+Forfeiture, Cancellation, Community Service and Other Orders
+(109) [1] The Court, before which a person is prosecuted for an offence against this Law or any regulations made there under, may, in addition to any other order: [a] upon the conviction of the accused; or [b] if it is satisfied that an offence was committed notwithstanding that no person has been convicted of the offence. [2] Order that the substance, equipment and appliance used in the commission of the offence be forfeited to the Agency and be disposed of as the court directs; [3] In making an order under subsection [1], the court may also order that the cost of disposing of the substance, equipment and appliance referred to in subsection [1], be borne by the accused; [4] The Court may further order the cancellation of any license, permit or other authorization given under this Law and to which the offence relates; [5] The Court may, in addition to any fine it may impose upon an accused person, require him to do community work, which promotes the protection of the environment. [6] The Court may also issue an environmental restoration order against the accused in accordance with Part VIII of this Law.
+
+PART XIII - MISCELLANEOUS 
+
+Regulations
+(110) [1] The Legislature may on the recommendations of the Agency and upon consultation with the Line Ministries, make regulations providing for matters that are required or permitted by this Law to give full effect to the purposes and objectives of this Law; [2] Regulations made under subsection [1] shall include: [a] The establishment of the Environmental Court; [b] Civil procedure for suits under section 5 of the Law; [c] Provide for the issue, amendment and revocation of any license issued under this Law; [d] Provide for fees and levies to be charged under this Law; [e] Prescribe anything required or permitted to be prescribed under this Law; [f] Provide for the protection of any particular species of fauna and flora; and [g] Provide for any regulations may be made under this Law.
+
+Rules of Construction
+(111) [1] The provisions of this Law shall be liberally construed to promote the sustainable protection and management of the environment and natural resources of Liberia; [2] References to sections in this Law or to other laws of Liberia shall mean those sections or other laws as they shall be amended from time to time; [3] Unless the context otherwise requires, words importing the singular include plural and vice versa, and words importing gender or neuter include both genders and the neuter; and [4] The word "including" means including but not limited to the items listed as included.
```

Comment:
Please replace:

* `Liberia; [2]` with `Liberia. [2]`
* `time; [3]` with `time. [3]`
* `gender or neuter` with `gender or neutral`
* `genders and the neuter; and [4]` with `genders and the neutral. [4]`